src/backends/caffe/caffelib.cc

```diff
@@ -187,6 +188,153 @@ namespace dd
 	caffe::ReadProtoFromTextFile(dest_deploy_net,&deploy_net_param);
 
 
+    if (this->_loss == 1)
```

Comment:
Can we hold the loss into a string instead, thus paving the way to using more custom losses via API ?

Reply:
done